lib/evader/view.js

```diff
@@ -214,10 +216,10 @@ Evader.extend(
 		};
 
 		const eventListeners = {
-			register(target, event, listener, options = { removeListener: false }) {
+			register(target, event, listener, options = { removeListener: false, eventOptions: null }) {
 				return options.removeListener
 					? target.removeEventListener(event, listener)
-					: target.addEventListener(event, listener);
+					: target.addEventListener(event, listener, eventOptions);
```

Comment:
It should be `options.eventOptions` in this line, instead of just `eventOptions`.